### DISCUSSION TITLE
如何自行本地部署faster-whisper模型

### DISCUSSION THREAD

**Discussion by MrFutureV** [Ayanaminn/N46Whisper#55] [Q&A]
好像用国外大神开发的whisper desktop不支持直接导入这个模型，有什么别的方法可以部署捏？
![QQ截图20230519212515](https://github.com/Ayanaminn/N46Whisper/assets/46783891/9edc39cd-77cf-4464-92f3-8cdfaa691e14)

**Comment by Nekofoxmiu** [comment]
https://github.com/Softcatala/whisper-ctranslate2
考慮這個專案吧
